packages/ui/src/lib/adapters/zkitter/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { ec } from 'elliptic'
+import { Strategy, ZkIdentity } from '@zk-kit/identity'
+import { generateIdentity } from 'zkitter-js'
+import type { Signer } from 'ethers'
+
+// FIXME: help! Would be nice to have this exposed
+import { sha256, signWithP256 } from 'zkitter-js/utils/crypto'
```

Comment:
@0xtsukino would be awesome if this could be exported (can be directly from core `zkitter-js` instead of 'zkitter-js/utils/crypto')